arrows/qt/widgets/LoggerView.cxx

```diff
@@ -0,0 +1,70 @@
+// This file is part of KWIVER, and is distributed under the
+// OSI-approved BSD 3-Clause License. See top-level LICENSE file or
+// https://github.com/Kitware/kwiver/blob/master/LICENSE for details.
+
+#include "LoggerView.h"
+
```

Comment:
Pedantic: `foo.h` and `ui_foo.h` are both in the "headers specific to this file" group.